app/pages/home-for-user/generic-section.jsx

```diff
@@ -21,7 +22,7 @@ const HomePageSection = React.createClass({
             }}
           ></i>
           <span className="home-page-section__header-label">{this.props.title}</span>
-          <a href="#" className="secret-button" title="Close this section" aria-label="Close this section">×</a>
+          <Link to="#" className="secret-button" title="Close this section" aria-label="Close this section">x</Link>
```

Comment:
This was changed from a simple anchor tag because the section would not close once navigating back to the homepage from another link. The `Link` component resolved this issue.